### ISSUE TITLE
[data-grid]: focus and keyboard navigation also includes the header row

### ISSUE THREAD

**Issue by YonatanKra** [microsoft/fast#6622]
When focusing in the data-grid, the header row's cells are focusable and navigatable:
![image](https://user-images.githubusercontent.com/6459899/215243653-0d613254-f6db-4ced-a0ca-879a5f005d40.png)

According to W3C example, it is not so:
https://www.w3.org/WAI/ARIA/apg/example-index/grid/dataGrids.html

Is the discrepancy intended?

**Comment by scomea** [comment]
The "Minimal Data Grid" sample says "Since the header cells do not provide any functionality, they are not focusable.".  The following example "Sortable Data Grid With Editable Cells" does have focusable headers which support the sorting functionality.

![image](https://user-images.githubusercontent.com/7649425/222086341-dc40c371-4bd3-4e4f-af22-36f034f7bdc0.png)

I think focusable headers is a useful feature, but I'd agree that there should be an option to make header rows not focusable.

**Comment by YonatanKra** [comment]
I actually opened an issue with W3C.

They agreed a focusable header should be the go-to (we focused on a11y).

https://github.com/w3c/aria-practices/issues/2626

**Comment by scomea** [comment]
Ok, works for me.  So, unless someone wants to argue for the ability to have non-focusable cells/rows we can close?

**Comment by YonatanKra** [comment]
Yea. I actually implemented it (non-focusable headers) so if at any point in time it is needed, I can push a PR :)

**Comment by scomea** [comment]
> Yea. I actually implemented it (non-focusable headers) so if at any point in time it is needed, I can push a PR :)

More contributors is a good thing. :-)
